commands_test.go

```diff
@@ -16,3 +16,16 @@ func TestMonitorStatusResult(t *testing.T) {
 	assert(t, command.MilActive == true, "MIL was not active")
 	assert(t, command.DtcAmount == 127, "DTCs were not 127")
 }
+
+func TestService01Command(t *testing.T) {
+	service01cmd := &Part1Supported{
+		BaseCommand{SERVICE_01_ID, 0, 4, "supported_commands_part1"},
+		UIntCommand{},
+	}
```

Comment:
How come we are constructing this type manually and not using the constructor `NewPart1Supported?

Reply:
good catch. Thanks